app/Http/Transformers/ReportbackTransformer.php

```diff
@@ -16,10 +16,12 @@ class ReportbackTransformer extends TransformerAbstract
     public function transform(Post $post)
     {
         $signup = $post->signup;
+
         $result = [
             'id' => $post->id,
             'status' => $post->status,
             'caption' => $post->caption,
+            'event_id' => $post->events[0]->id,
```

Comment:
Ah right ! Now with the `POST /reactions` endpoint we use `post_id` instead of `event_id`